pages/Home.tsx

```diff
@@ -90,6 +94,21 @@ export function Home() {
               onPress={switchBalanceState}
               className="w-full flex flex-col items-center justify-center gap-4"
             >
+              {wallets.length && (
```

Comment:
should this be `{wallets.length > 1 &&}` so it only shows if the user actually uses multiple wallets? - also won't render 0 if there are no wallets (I guess we don't see this page in that case though)